extensions/GlobalMessages/GlobalMessagesT.i18n.php

```diff
@@ -32,6 +32,7 @@
 	'tog-previewonfirst' => 'Show preview on first edit',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "tog-previewonfirst" 254 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)